src/Components/blog/BlogWall.js

```diff
@@ -0,0 +1,75 @@
+import React,{useState} from 'react';
+import { getLocalStorage } from '../api/localStorageApi';
+import BlogPostEditModal from './BlogPostEditModal';
+import BlogPost from './BlogPost';
+import PropTypes from 'prop-types';
+
+const BlogWall = ({authorization, updatePostsData}) => {
+
+    const [toggleEditModal, setToggleEditModal] = useState(false);
+    const [postData, setPostData] = useState(null);
+    const [id, setId] = useState(null);
+    const posts = getLocalStorage('posts');
+    const date = new Date().toJSON().slice(0,10);
+
+    const handleClick = () => {
+        if (authorization.login === 'admin'){
+            localStorage.removeItem('posts');
+            updatePostsData([]);
+        } else {
+            alert('Удалить все посты может только администратор');
+        }
+    }
```

Comment:
```suggestion
    const handleClick = () => {
            localStorage.removeItem('posts');
            updatePostsData([]);
    }
```

можешь удалить логику и просто не отрисовывать кнопку удаления всех постов если пользователь не админ